ember/app/components/settings-panels/profile.js

```diff
@@ -50,14 +50,15 @@ export default Ember.Component.extend(Validations, {
   speedUnits: ['m/s', 'km/h', 'kt', 'mph'],
   liftUnits: ['m/s', 'kt', 'ft/min'],
   altitudeUnits: ['m', 'ft'],
-
   distanceUnit: computedUnit('distanceUnits', 'distanceUnitIndex'),
   speedUnit: computedUnit('speedUnits', 'speedUnitIndex'),
   liftUnit: computedUnit('liftUnits', 'liftUnitIndex'),
   altitudeUnit: computedUnit('altitudeUnits', 'altitudeUnitIndex'),
 
   unitsPresets: ['custom', 'european', 'british', 'australian', 'american'],
 
+  uploadPermissionClub: null,
```

Comment:
could you put that further up below `altitudeUnitIndex`? that way all the inputs of the component are somewhat grouped together.